persist/src/main/resources/db/migration/V1__Initial_Import.sql

```diff
@@ -42,6 +141,62 @@ BEGIN
 END;
 $$ language 'plpgsql';
 
+DROP TRIGGER IF EXISTS countries_trig_set_updated_at ON countries;
+CREATE TRIGGER countries_trig_set_updated_at
+    BEFORE UPDATE
```

Comment:
How do we set these columns for DELETE operations?

Reply:
`created_at`, `updated_at` should never be manually set. On `DELETE`, if we want to soft-delete, we should just set `deleted_at` to `NOW()` and for those tables query `WHERE deleted_at IS NULL`